Algebraic Topology/algebraic-topology.tex

```diff
@@ -1,5 +1,5 @@
 \documentclass[11pt,leqno,oneside]{amsart}
-\usepackage[margin=1in]{geometry}
+%\usepackage[margin=1in]{geometry}
```

Comment:
I changed the margins to the default because I feel 1in margins put a lot of text on one page.